src/validator.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Validates ccss against custom defined rules.
+ */
+function validateCSS(ccss) {
+  _validateNotFillFile(ccss);
+}
+
+/**
+ * Throw an exception if the ccss has colored file makers.
+ * Tangram cannot handle painting a image causing a bad result.
+ */
+function _validateNotFillFile(ccss) {
+  if ((ccss.indexOf('marker-fill') >= 0) && (ccss.indexOf('marker-file') >= 0)) {
```

Comment:
Using `indexOf` might give us some false positives. For example, when using the following CartoCSS (with conditionals):

```
#layer {
  marker-width: 7;
  marker-fill: #FFB927;
  marker-fill-opacity: 0.9;
  marker-allow-overlap: true;
  marker-line-width: 1;
  marker-line-color: #FFF;
  marker-line-opacity: 1;
}

#layer[name='Madrid'] {
  marker-width: 47;
  marker-file: url('https://s3.amazonaws.com/com.cartodb.users-assets.production/maki-icons/triangle-stroked-18.svg');
  marker-fill-opacity: 0.9;
  marker-allow-overlap: true;
  marker-line-width: 1;
  marker-line-color: #FFF;
  marker-line-opacity: 1;
}
```